types/supercluster/index.d.ts

```diff
@@ -4,96 +4,120 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import * as GeoJSON from 'geojson';
+import * as GeoJSON from "geojson";
 
 export as namespace supercluster;
 
-export interface Options {
-    /**
```

Comment:
@kl4kennylee81 What happened with the indentation?

![image](https://user-images.githubusercontent.com/550895/32411365-095b7efe-c1af-11e7-9509-383593d3185e.png)

Reply:
sorry my text editor had different tab/space setting and I committed without looking it over closely. My apologies.